api/client/stack/opts.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 func addBundlefileFlag(opt *string, flags *pflag.FlagSet) {
-	flags.StringVarP(
+	flags.StringVar(
 		opt,
-		"bundle", "f", "",
+		"bundle", "",
```

Comment:
Yeah, still wondering if we should change this to `--file`

@dnephin @aanand wdyt?